DigitalPassaport/DigitalPassaport/Infactuture/Networking/NetworkingService.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  NetworkingService.swift
+//  DigitalPassaport
+//
+//  Created by Milena Predic on 24.8.23..
+//
+
+import Foundation
+import Combine
+import Alamofire
+
+protocol NetworkingServiceProtocol {
+    func post(_ url: URL, parameters: [String: Any]) -> AnyPublisher<Data, APIError>
+}
+
+class NetworkingService: NetworkingServiceProtocol {
+    
+    func post(_ url: URL, parameters: [String: Any]) -> AnyPublisher<Data, APIError> {
+        return AF.request(url, method: .post, parameters: parameters)
```

Comment:
Don't use static AF, instead create Alamofire Session and use it for each API call